documentation/modules/ref-example-roles.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-tenant-guide.adoc
+// using master.adoc
+
+[id='ref-example-roles-{context}']
+= {ProductName} resources and {ProductName} example roles
+
+The following table describes the {ProductName} resources.
```

Comment:
I think we should split the table into two: resources that are of interest to the service administrator and those interest to the messaging tenant.